FlowCrypt/Functionality/Services/Compose Message Service/ComposeMessageService.swift

```diff
@@ -166,7 +166,7 @@ final class ComposeMessageService {
 
         let subject = contextToSend.subject ?? "(no subject)"
 
-        guard let myPubKey = try storage.getKeypairs(by: sender).map(\.public).first else {
+        guard let keyPair = try await keyService.getKeyPair(from: sender) else {
             throw MessageValidationError.missingPublicKey
         }
```

Comment:
We have some mess in the keyService and storage methods that's hard to exactly understand or explain. Once the other PR is merged, I'll look into this to clean it up (in this PR).

Reply:
what do you mean `other PR`?